src/hello_client.erl

```diff
@@ -165,12 +166,16 @@ handle_call(terminate, _From, State) ->
 %% @hidden
 handle_info({?INCOMING_MSG, Message}, State) ->
     incoming_message(Message, State);
+handle_info(?PING, State = #client_state{waiting_for_pong = true, keep_alive_interval = KeepAliveInterval}) ->
+    ?LOG_ERROR("There is no PONG answer on PING for ~p msec. Client will be restarted.", [KeepAliveInterval]),
```

Comment:
@thz 1) we are restarting client process. if would be more accurate, we are stopping client process and supervisor will restart it if it possible. Maybe we should write "Client will be stopped" and have log message when client starts? 2) Why we should wait for 7 seconds if KeepAliveInterval is 5? When KeepAliveInterval is expired we stop process (actually, we try to send new PING message and realize that there is no PONG and we stop process) see [hello_client.erl](https://github.com/travelping/hello/blob/master/src/hello_client.erl#L172). Of course, these values more approximately but not like 5 and 7 seconds.